frame/ethereum/src/lib.rs

```diff
@@ -769,14 +780,19 @@ impl<T: Config> Pallet<T> {
 		}
 	}
 
-	pub fn inject_emulated_transaction_result(source: H160, logs: Vec<MaybeMirroredLog>) {
```

Comment:
There is no breaking change, as we already only pass zero address to this method
https://github.com/UniqueNetwork/unique-chain/blob/develop/pallets/evm-coder-substrate/src/lib.rs#L199